app/src/main/res/values/strings.xml

```diff
@@ -1103,4 +1105,31 @@
     <string name="search_edit_custom_engine_success_message">Saved %s</string>
     <!-- Text shown when a user successfully deletes a custom search engine -->
     <string name="search_delete_search_engine_success_message">Deleted %s</string>
+
+    <!-- Add-on strings -->
+    <string name="install_add_on_content_description">Install Add-on</string>
```

Comment:
This strings are now part of [feature-addon](https://github.com/mozilla-mobile/android-components/pull/5337)